File: code/ChicType/app/src/main/java/org/depinfo/chictype/network/http/RetrofitUtil.java
package org.depinfo.chictype.network.http;

import retrofit2.Retrofit;
import retrofit2.converter.scalars.ScalarsConverterFactory;

public class RetrofitUtil {
    public static Service get(){
        Retrofit retrofit = new Retrofit.Builder()
                .addConverterFactory(ScalarsConverterFactory.create())
                .baseUrl("https://4n6.azurewebsites.net/")
                .build();

        return retrofit.create(Service.class);
    }
}

File: code/ChicType/app/src/main/java/org/depinfo/chictype/network/http/RetrofitUtil.kt
package org.depinfo.chictype.network.http

import retrofit2.Retrofit
import retrofit2.converter.scalars.ScalarsConverterFactory

object RetrofitUtil {
    fun get(): Service {
        val retrofit = Retrofit.Builder()
            .addConverterFactory(ScalarsConverterFactory.create())
            .baseUrl("https://fourn6-mobile-prof.onrender.com/")
            .build()

        return retrofit.create<Service>(Service::class.java)
    }
}
